web_src/css/repo/home-file-list.css

```diff
@@ -22,39 +33,63 @@
   display: contents;
 }
 
-#repo-files-table .repo-file-item:hover > .repo-file-cell {
+#repo-files-table .repo-file-item:hover > .repo-file-cell,
+#repo-files-table .parent-link:hover {
   background: var(--color-hover);
 }
 
 #repo-files-table .repo-file-line,
 #repo-files-table .repo-file-cell {
-  border-top: 1px solid var(--color-light-border);
-  padding: 6px 10px;
+  border-top: .5px solid var(--color-light-border);
+  border-bottom: .5px solid var(--color-light-border);
+  padding: 8px 10px;
 }
 
 #repo-files-table .repo-file-line:first-child {
   border-top: none;
+  border-radius: var(--border-radius) var(--border-radius) 0 0;
+}
+
+#repo-files-table .repo-file-item:last-child .repo-file-line,
+#repo-files-table .repo-file-item:last-child .repo-file-cell {
+  border-bottom: none;
+}
+
+#repo-files-table .repo-file-item:last-child {
+  border-bottom: none;
+}
+
+#repo-files-table .repo-file-item:last-child .repo-file-cell:first-child {
+  border-bottom-left-radius: calc(var(--border-radius) - 1px);
+}
+
+#repo-files-table .repo-file-item:last-child .repo-file-cell:last-child {
+  border-bottom-right-radius: calc(var(--border-radius) - 1px);
 }
 
 #repo-files-table .repo-file-line {
   grid-column: 1 / span 3;
   display: flex;
   align-items: center;
   gap: 0.5em;
-  padding: 6px 10px;
 }
 
 #repo-files-table .repo-file-last-commit {
   background: var(--color-box-header);
 }
 
 #repo-files-table .repo-file-cell.name {
-  max-width: 300px;
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
 }
 
+@media (max-width: 767.98px) {
```

Comment:
The same `@media` selectors could be merged together to make it easier to maintain.

Reply:
Yes, but I prefer for a certain selector to keep media queries close to the non-query selector, so that when editing one does not forget about changing the media query content.